apps/nextjs-website/src/helpers/webinars.helpers.ts

```diff
@@ -0,0 +1,23 @@
+import { Webinar } from '@/lib/types/webinar';
+
+export const getFutureWebinars = (
+  webinars: readonly Webinar[]
+): readonly Webinar[] =>
+  webinars.filter(
+    ({ startDateTime, endDateTime }) =>
+      (!startDateTime && !endDateTime) ||
+      (endDateTime && new Date(endDateTime).getTime() > new Date().getTime())
+  );
+
+export const getPastWebinars = (
+  webinars: readonly Webinar[]
+): readonly Webinar[] =>
+  webinars
+    .filter(
+      ({ endDateTime, startDateTime }) =>
+        endDateTime &&
+        startDateTime &&
+        new Date(endDateTime).getTime() <= new Date().getTime()
+    )
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    .sort((a, b) => (a.startDateTime! < b.startDateTime! ? 1 : -1));
```

Comment:
Avoid using the `!` operator. While it may seem safe due to the presence of a filter, consider the scenario where someone alters the filter condition, removing the checks on `endDateTime` and `startDateTime`. In such a case, no failing tests would occur, the code would compile, but the sorting would not be handled properly